packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -147,6 +149,9 @@ class SemanticsData extends Diagnosticable {
   ///
   /// The reading direction is given by [textDirection].
   final String hint;
+  
+  /// The name of a widget subtree.
```

Comment:
This and elsewhere need a stronger doc comment. What's the route name used for? What do TalkBack and VoiceOver do with it? When should this be set?